### DISCUSSION TITLE
DAI 2024-2025 - Practical work 2 - Carbonara Nicolas, Surbeck Léon

### DISCUSSION THREAD

**Discussion by Bananonymous** [heig-vd-dai-course/heig-vd-dai-course#496] [Show and tell]
For Practical work 2, we will develop a simple messaging system. The project will utilize client concurrency to manage multiple clients connecting simultaneously. It will allow the creation of private discussion rooms for user communication. This project will (probably) use TCP & UDP. Additionally, we aim to integrate encryption features to ensure secure exchanges if time permits.

[Github Repo](https://github.com/Bananonymous/DAI-Labo2)

**Comment by ludelafo** [comment]
Looks good to us!

**Comment by Syseria** [comment]
https://github.com/Bananonymous/DAI-Labo2

Hash: 686808ac6f625f9313a61bef567424cbfa9af750
